src/main/msp/msp.c

```diff
@@ -1748,8 +1748,8 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         // Added in MSP API 1.42
 #if defined(USE_GYRO_DATA_ANALYSE)
         sbufWriteU8(dst, 0); // DEPRECATED 1.43: dyn_notch_range
-        sbufWriteU8(dst, gyroConfig()->dyn_notch_width_percent);
-        sbufWriteU16(dst, gyroConfig()->dyn_notch_q);
+        sbufWriteU8(dst, gyroConfig()->dyn_notch_count);
+        sbufWriteU16(dst, gyroConfig()->dyn_notch_bandwidth_hz);
```

Comment:
These should be added to the end instead. A client may not be aware of these changes and the user can end up changing the wrong parameters. Look at how it was done when ```dyn_notch_range``` was removed and ```dyn_notch_max_hz``` was added.

Reply:
Am I correct in assuming that the next MSP API version is 1.45?

Reply:
Yes, but if this is going into 4.3 the MSP API version is 1.44

Reply:
If it's not going into 4.3 in time, I will adjust the version number in the comments